docs/App Services/index.md

```diff
@@ -0,0 +1,94 @@
+
+### Supported App Service Types?
+
+The defined and supported app types as of writing this guide are as follows (linked to RPC spec data definitions):
+
+- [Media](https://github.com/smartdevicelink/rpc_spec#mediaservicedata)
+- [Weather](https://github.com/smartdevicelink/rpc_spec#weatherdata)
+- [Navigation](https://github.com/smartdevicelink/rpc_spec#navigationservicedata)
+
+For the up-to-date list, the [`AppServiceType`](https://github.com/smartdevicelink/rpc_spec#appservicetype) enum in the RPC spec can be used as reference.
+
+#### Future App Service Types
+
+The current list of app services is by no means the only services that will be added. These services were just the first round of types to be added for their familiarity and usefulness in terms of creating an in-car user experience. Also, as the original proposal points out, as new services become available even older modules with old versions of SDL Core can still manage the publishing and subscription of those services using the new types. While the module will likely not be able to make use of the new service, other SDL connected applications can access them as they become defined in the RPC spec and available from installed apps.
+
+
+### Who is an App Service Provider?
+
+An app service provider is the originator of a service that provides updates of related service data. For example, a connected music player application can be a Media Service or an embedded Navigation app can be a Navigation Service. The app service provider can live on either side of the SDL connection, core or application library side.
+
+It is up to each app to decide if they wish to expose their service or not. It is not a requirement for apps to do so. 
+
+
+### Who is an App Service Consumer?
+
+An app service consumer retrieves and possibly subscribes to updates of service data that is provided through an app service provider. For example, the module side with SDL Core could use the media app service data from a music player app to place that information outside the standard templates provided. A connected SDL app could navigation app service data from the embedded navigation app service to determine the best place to fill up the car with gas. 
+
+#### How does a module/Core use app service data as a consumer?
+
+The main use case for using app service data on the module as a consumer is to be able to present that data in a specific and predictable way. The module can maintain branding and their user experience by taking the service data and placing it outside the generic template boxes where they might populate that data with their embedded data normally.
+
+##### Example with Generic HMI
+
+The following is an example of populating specific areas of the HMI with app service data.
+
+###### Collapsed View
```

Comment:
Disagree with removing this.

Reply:
The reason I recommended removing this and the below was because the level-six header looks _terrible_ on the dev portal. I figured the context of the paragraph made the image clear enough. If you don't want to remove it, then I'd recommend either altering how the level-six header is styled or altering the level structure.